src/Symfony/Bundle/SecurityBundle/RouteParamsMatcher.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+/*
```

Comment:
miss one *

Reply:
I copied this part from another file so I don't think it's wrong.

Reply:
You’re right sorry 👍🏽